modules/configure-acs-kube-security-platform.adoc

```diff
@@ -0,0 +1,45 @@
+[id="configure-acs-kube-security-platform"]
+= Configuring the {product-title} platform
+:context: configure-acs-kube-security-platform
+
+Create a new integration in the {product-title} platform by using the email plugin notifier.
+
+The following steps represent the workflow to create a new integration:
+
+. Navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Plugins* section, select *Email*.
+. Select the *New Integration* icon.
+. Enter a name for your email integration in the *Integration Name*.
+. For *Email Server*, enter the address of your email server.
+. Enter *Username* and *Password* to use for authentication.
+. (Optional) For *FROM*, specify the *email FROM header*.
+. For *SENDER*, specify the *email SENDER header*.
+. Enter the email address on which to send the notifications in the *Default Recipient* box or enter an annotation key for *Annotation Key For Recipient*.
+. (Optional) Turn on the *Connect Without TLS (Unencrypted)* toggle to send email without TLS. You should not disable TLS unless you are using StartTLS.
++
+[NOTE]
+====
+Use TLS for email notifications. Without TLS, all email is sent unencrypted.
+====
+
+. (Optional) To use StartTLS, either select *Login* or *Plain* for the *Use STARTTLS (Requires TLS To Be Disabled)* drop-down.
++
+[IMPORTANT]
+====
+Because un-encrypted credentials are sent to the email server during authentication, do not use StartTLS.
+
+* StartTLS with the *login* parameter sends authentication credentials in a `*base64*` encoded string.
+* StartTLS with the *plain* parameter sends authentication credentials to your mail relay in plain text.
+====
+
+To send email through an annotation in your deployment:
```

Comment:
This isn't really sending email _through_ an annotation -- this is using an annotation to dynamically determine the e-mail recipient based on an annotation.  This needs a lot more context so that customers know what they can accomplish and how.

Additionally, the annotation can be on either the deployment _or_ the namespace.

Reply:
> This isn't really sending email _through_ an annotation -- this is using an annotation to dynamically determine the e-mail recipient based on an annotation. This needs a lot more context so that customers know what they can accomplish and how.
> 
> Additionally, the annotation can be on either the deployment _or_ the namespace.

@gaurav-nelson  when you get a moment, let's have a chat about this one. I have already made the other two and got a /LGTM. Thanks!